jobrunrpro/src/main/java/org/axonframework/extensions/jobrunrpro/util/JobRunrProUtils.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2010-2023. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.extensions.jobrunrpro.util;
+
+import org.jobrunr.jobs.Job;
+import org.jobrunr.jobs.states.StateName;
+import org.jobrunr.scheduling.JobScheduler;
+import org.jobrunr.storage.JobSearchRequest;
+import org.jobrunr.storage.StorageProvider;
+
+import java.util.List;
+
+import static org.jobrunr.storage.JobSearchRequestBuilder.aJobSearchRequest;
+import static org.jobrunr.storage.PageRequest.ascOnUpdatedAt;
+
```

Comment:
I'm missing JavaDoc throughout this class.

Reply:
Added